### DISCUSSION TITLE
hide top border color on mac

### DISCUSSION THREAD

**Discussion by aasutossh** [kovidgoyal/kitty#5737] [Q&A]
<img width="1042" alt="image" src="https://user-images.githubusercontent.com/18498932/205559174-1ad67863-5744-461e-8ea9-7cf820e34a78.png">
Is there any option in the configs?

**Comment by aasutossh** [comment]
`hide_window_decorations titlebar-only` is enabled.

**Comment by page-down** [comment]
Do you need to hide the outer border of the OS window (don't like that "color")?
That is a system decision based on the contrast between the edge and the background, kitty has no control over that.

You have found the configuration option, just specify it as `yes` to hide the border.

```conf
hide_window_decorations yes
```

UPDATE:
Currently the only way to get the system to draw the border without adding highlighting (the white line you see) is to have the window with the opacity property enabled.
You can use the following configuration for now (kitty 0.26.5):

```conf
hide_window_decorations titlebar-only
background_opacity 0.98
```

**Comment by aasutossh** [comment]
Was using alacritty before, it didn't have that top border. I thought kitty was doing something.

**Reply by aasutossh** [reply]
I tried looking for it on the internet, unable to find a solution.
Found out about a MacForge plugin, I can't even boot into recovery partition so that's that.
[source](https://stackoverflow.com/a/67433400/11690968)

**Reply by page-down** [reply]
> ... it didn't have that top border ...

It has a title bar by default, so I don't know what you are referring to.

> ... unable to find a solution ...
> "How to disable shadows ..."

If you configured `decorations: none` in the other terminal, then the equivalent kitty configuration (`hide_window_decorations yes`) has been pointed out in the reply above.

**Reply by aasutossh** [reply]
When I used
```
window:
  decorations: buttonless
``` 
in alacritty there was no top border. (and I can still move & resize the window)
<img width="1054" alt="image" src="https://user-images.githubusercontent.com/18498932/205585652-ef76018c-7318-495f-b261-a9f4dcb3cde6.png">

see the difference between alacritty and kitty.

Setting `hide_window_decorations yes` won't let me move, resize the (kitty) window.

**Reply by page-down** [reply]
@aasutossh
Since you provided more information, I read the source code of alacritty and found that **its window is always opaque disabled** (window is transparent even if opacity is 1.0).
I've updated the reply above (`background_opacity 0.98`).
The current drawback is that the window is not completely opaque, perhaps losing the deeper shadows (with only lighter shadows), and I might submit a PR to suggest a new configuration value to support this style.
